region/crateria/east.json

```diff
@@ -2662,6 +2662,153 @@
+                  ]
+                },
+                {
+                  "name": "Full Room Shinespark Plus SpaceJump",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      {"and": [
+                        {"enemyDamage": {
+                          "enemy": "Choot",
+                          "type": "contact",
+                          "hits": 1
+                        }},
+                        "canNeutralDamageBoost",
+                        "canBounceBall"
+                      ]},
+                      {"and": [
+                        {"or": [
+                          "canPseudoScrew",
+                          "ScrewAttack",
+                          "Wave+Plasma"
+                        ]},
+                        "canDodgeWhileShooting",
+                        "canMockball"
+                      ]}
+                    ]},
+                    {"canComeInCharged": {
+                      "framesRemaining": 180,
+                      "fromNode": 1,
+                      "shinesparkFrames": 123,
+                      "excessShinesparkFrames": 5,
+                      "inRoomPath": [1,3],
+                      "unusableTiles": 1
+                    }},
+                    "canShinechargeMovementComplex",
+                    "canSpaceJumpWaterBounce",
+                    "canCarefulJump",
+                    {"or": [
+                      {"and": [
+                        "HiJump",
+                        "canSpaceJumpWaterEscape"
+                      ]},
+                      "canWalljump"
+                    ]}
+                  ],
+                  "note": [
+                    "Enter the room with a shinespark stored, and use it on the lowest part of the ramp to cross the room and reach a raised ledge on the right side wall.",
+                    "From there, use SpaceJump to escape the water.",
+                    "Carry the shinespark to the bottom of the ramp either with a bounce ball which takes damage from the Choot, or use a mockball and kill the Choot."
+                  ]
+                },
+                {
+                  "name": "East Ocean Shinespark Water Escape",
+                  "notable": true,
+                  "requires": [
+                    {"or": [
+                      {"and": [
+                        {"enemyDamage": {
+                          "enemy": "Choot",
+                          "type": "contact",
+                          "hits": 1
+                        }},
+                        "canNeutralDamageBoost",
+                        "canBounceBall"
+                      ]},
+                      {"and": [
+                        {"or": [
+                          "canPseudoScrew",
+                          "ScrewAttack",
+                          "Wave+Plasma"
```

Comment:
I would assume the kill strats take a lot more shinecharge frames, does it make this strat particularly difficult, since it's already very tight?

Reply:
The kill strats are just as precise.  The hardest part is still unmorphing so that you are in position to spark at the end of the runway.